runtime-testsuite/test/org/antlr/v5/test/runtime/java/api/TestVisitors.java

```diff
@@ -30,7 +26,7 @@ public class TestVisitors {
 	@Test
 	public void testVisitTerminalNode() {
 		String input = "A";
-		VisitorBasicLexer lexer = new VisitorBasicLexer(new ANTLRInputStream(input));
+		VisitorBasicLexer lexer = new VisitorBasicLexer(CharStreams.fromString(input));
```

Comment:
It looks like unrelated changes.

Reply:
It relates to housekeeping, see above.
Since I had to revisit many files, I took the opportunity.

Reply:
Please try to place different changes to different PR next time. It confuses reviewing process especially considering there are a lot of changes.

Reply:
Yeah I would have if I had thought about it before engaging this split. Then as part of the split I find myself having to decide what to do with this obsolete stuff, and it's obviously better to prune it